docs/framework/configure-apps/file-schema/application-settings-schema.md

```diff
@@ -16,6 +16,9 @@ Application settings allow a Windows Forms or ASP.NET application to store and r
 
 By default, application settings in a Windows Forms application uses the <xref:System.Configuration.LocalFileSettingsProvider> class, which uses the .NET configuration system to store settings in an XML configuration file. For more information about the files used by application settings, see [Application Settings Architecture](/dotnet/desktop/winforms/advanced/application-settings-architecture).
 
+> [!NOTE]
+> Most configuration sections defined by .NET Framework don't function in .NET 6+ and .NET Core versions. <xref:System.Configuration.ConfigurationManager> is provided only for compatibility. Instead, consider removing the usage in app.config and calling the corresponding API, if available, to make the same setting.
```

Comment:
I think this should be expanded a bit to say that `app.config` was replaced by [`appsettings.json`](https://learn.microsoft.com/en-us/dotnet/core/porting/modernize#appconfig), and that the [configuration api](https://learn.microsoft.com/en-us/dotnet/core/extensions/configuration) is the modern way to manage your apps config and settings.

Reply:
@adegeo I've made some changes per your comments - let me know if these are OK. 

There's an outstanding comment from @ericstj  in the original issue suggesting that we include the configuration sections along with the corresponding APIs. I'm not sure how to address that request - I haven't found a neat one-to-one listing of config to APIs to use. Suggestions appreciated.

cc @gewarren cc @MeeraDi

Reply:
The old sections are listed in C:\Windows\Microsoft.NET\Framework\v4.0.30319\Config\machine.config
We'd have to go through section-by-section to call out if it was read at all by .NET Core and if there is a replacement API.

Reply:
@adegeo Can you sign off on the changes I made per your comments - I'd like to close this thread.

I'd be happy to add a list of APIs, if we can compile it, but I'm still not sure how to compile it. Unless there's a clear path forward, we'll close the issue on the current changes.

cc @MeeraDi